sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -222,7 +223,8 @@ PROGRAM WRFPOST
         read(5,120) grib
         if (me==0) print*,'OUTFORM= ',grib
         if(index(grib,"grib") == 0) then
-          grib='grib1'
+!          grib='grib1'
```

Comment:
Maybe include an informative print statement for the log file to inform user of grib2 only support incase they have tried to run with grib1 output.